airbyte-integrations/connectors/source-trustpilot/source_trustpilot/manifest.yaml

```diff
@@ -0,0 +1,339 @@
+version: 0.71.0
+type: DeclarativeSource
+
+definitions:
+  oauth_authenticator:
+    type: OAuthAuthenticator
+    client_id: '{{ config[''credentials''][''client_id''] }}'
+    client_secret: '{{ config[''credentials''][''client_secret''] }}'
+    refresh_token: '{{ config[''credentials''][''refresh_token''] }}'
+    token_refresh_endpoint: 'https://api.trustpilot.com/v1/oauth/oauth-business-users-for-applications/accesstoken?response_type=token'
+    grant_type: refresh_token
+    access_token_name: access_token
+
+  apikey_authenticator:
+    type: ApiKeyAuthenticator
+    header: apikey
+    api_token: "{{ config['client_id'] }}"
+
+  selective_authenticator:
+    type: SelectiveAuthenticator
+    authenticator_selection_path: [ "credentials", "auth_type" ]
+    authenticators:
+      oauth: "#/definitions/oauth_authenticator"
```

Comment:
let's keep this as oauth2.0 for backward compatibility

Reply:
Done!